### PR TITLE
feat(positions): Add final deadline field statutory positions

### PR DESCRIPTION
**Pull request by LeonVreling**
Sorry that the branch name is weird, my Git does not want to work with me tonight 😅

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -15,7 +15,5 @@
   </head>
   <body>
     <div id="app"></div>
-
-    <script data-jsd-embedded data-key="21430358-44e3-4787-8d72-e23b56714c03" data-base-url="https://jsd-widget.atlassian.com" src="https://jsd-widget.atlassian.com/assets/embed.js"></script>
```
Comment (serge1peshcoff) [inline public/index.html:19]: this is in master already right?

Comment (WikiRik) [reply public/index.html:19]: Yeah, but Leon is working from that same branch because Git was acting up for him

**Review by WikiRik — APPROVED** [review]

File: src/views/statutory/PositionsList.vue [hunk 2]
```diff
@@ -221,6 +225,10 @@ export default {
     },
     requirementsIsSet () {
       return (this.selectedPosition.requirements !== null && this.selectedPosition.requirements !== '')
+    },
+    // TODO: This is a temporary solution, in the future, we want to display either the "first" deadline OR the force_end deadline
```
Comment (WikiRik) [inline src/views/statutory/PositionsList.vue:229]: When we fix this, we should also think about what is good UI/UX wise. And that the final deadline might be earlier if enough people applied

**Pull request merged**